dime-coding-standards/checklists/tex/hfcs_checklist.tex

```diff
@@ -0,0 +1,282 @@
+\definecolor{fontcolor}{HTML}{7A0569}
+
+\titleformat{\section}%
+{\Large\rmfamily\bf\color{fontcolor}}% format applied to label+text
+{\llap{\colorbox{fontcolor}{\parbox{1.5cm}{\hfill\huge\color{fontcolor}\thesection}}}}% label
+{2pt}% horizontal separation between label and title body
+{}% before the title body
+[]% after the title body
+
+\titleformat{\subsection}%
+{\large\rmfamily\color{fontcolor}}% format applied to label+text
+{}% label
+{1.5pt}% horizontal separation between label and title body
+{}% before the title body
+[]% after the title body
+
+\newcommand{\dimeCheckBox}[1]{\CheckBox[height=0.01cm, width=0.4cm, bordercolor=gray]{#1}}
+\newcommand{\dimeTextField}[3]{\TextField[name=#1, height=0.3cm, width=#2, bordercolor=gray]{#3}}
+
+
+\newcommand{\titleBox}[1]{
+	\begin{tcolorbox}
+		[colframe = fontcolor,
+		colback = fontcolor,
+		sharp corners,
+		halign = flush center,
+		valign = center,
+		height = 0.3\textwidth,
+		after skip = 1cm]
+		#1
+	\end{tcolorbox}
+}
+
+\begin{document}
+	\begin{fullwidth}
+		
+		\titleBox{
+			\textcolor{white}{\LARGE{\textbf{DIME Analytics \\ Peer Code Review - High-Frequency Checks Checklist}} \\
+				\Large\textbf{{v1.0}}}
+		}
+
+  \dimeTextField{time}{2.5 cm}{Reviewer:} 
+
+        \dimeTextField{time}{2.5 cm} {Coder:}
+		
+		\textbf{NOTE:} Make sure that fill out this checklist \textbf{ONLY IF} your partner's submission includes request to review \textbf{high-frequency checks}. 
+		
+		\section*{High-frequency checks}
+  This checklist lists important factors to consider while reviewing your code review partner's \textbf{high-frequency checks}. Please fill this checklist, and submit it as an attachment when you submit \href{https://survey.wb.surveycto.com/collect/code_review_summary?caseid=}{this detailed form.}
+
+  \hfill \break 
+  \textbf{Note:} We are piloting this checklist as part of improving the code review process. The questions in this list are therefore \textbf{not required}, and we welcome your feedback on the points listed below.
+
+  \subsection*{General}
+		
+		\begin{itemize}
+			\setlength\itemsep{-0.1em}
+			\item[] \dimeCheckBox{The data shared does not contain any personally-identifying information (PII).}
+
+                \item[] \dimeCheckBox{The code creates a tracking sheet (or dashboard) to track progress of the survey.}
+        \end{itemize}
+
+ \hfill \break 
+The following sections cover various aspects of the \textbf{contents of the high-frequency checks} included in the package, but not all may apply to the high-frequency checks you are reviewing.
+
+\subsection*{Survey progress}
+Tracking \textbf{survey progress} involves checks for the following:
+            \begin{itemize}
+
+            \item[]\dimeCheckBox{\textbf{Response rates -}} 
+            The high-frequency checks contain/export:
+            
+            \begin{itemize}
+            
+                \item[] \dimeCheckBox{Table on number/share of surveys attempted, completed, and declined.}
```

Comment:
i don't think we need to specify format (table, bar graph, etc). So here the important poitn is that they are tracking number/share of surveys attempted/completed/declined but it doesn't have to be in a table. same for all items.